packages/icons/manifest.json

```diff
@@ -1030,6 +1040,11 @@
       "category": "Product and feature",
       "size": "24"
     },
+    {
```

Comment:
Where is this icon going to be used on the marketing website? Alongside the other card schemes?

Reply:
<img width="934" alt="Screenshot 2022-05-24 at 14 41 01" src="https://user-images.githubusercontent.com/11762408/170049530-bde04ad9-44ab-4c55-839e-b86d76e7d8f6.png">

this is going to be used alongside the other card schemes

Reply:
Then I'd add it to the _Card Schemes_ category. Icons in that category can deviate from our own design guidelines since they're usually branded.

Reply:
ok, thanks! I would only argue that maybe it is even better to put it in the _Payment method_ category, because the _Card scheme_ icons always follow the same formatting (round corners, grey border) which is not the case for the NFC payment, wdyt?

Reply:
after @amelako ended up moving it to the _Card Scheme_ category 👌